src/save_audio.rs

```diff
@@ -122,6 +127,8 @@ pub fn save_audio_file_to_disk(mut audio_bytes: Vec<u8>, device_config: DeviceCo
                     &location_timestamp,
                     "-metadata",
                     &location_accuracy,
+                    "-metadata",
+                    &sr,
```

Comment:
I think we should do the thing of rounding the sample_rate we get from the audio header to 48000, and if we're going to add that to the metadata, then we should give it a better name, like `sampleRate` – `sr` is a bit cryptic.